### PR TITLE
[Cassandra] Update as per best practices

### PR DESCRIPTION
**Pull request by harnish-crest**
- Enhancement

## What does this PR do?

This PR enhances the Cassandra integration with some of the best practices of developing integration packages.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/couchbase directory.
- Run the following command to run tests. 
> elastic-package test

### CODE REVIEW

**Review by kush-elastic — APPROVED** [review]
LGTM

**Comment by ManojS-shetty** [comment]
Lets make sure we always use vis by value: #3448

File: packages/cassandra/data_stream/log/fields/fields.yml [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
-    - name: log
-      type: group
-      fields:
-        - name: meta
-          type: keyword
-          description: Log meta infos like java stack_trace
-- name: log
-  type: group
-  fields:
-    - name: flags
```
Comment (ManojS-shetty) [inline packages/cassandra/data_stream/log/fields/fields.yml:13]: Can we know why flags field has been removed from here?

Comment (harnish-crest) [reply packages/cassandra/data_stream/log/fields/fields.yml:13]: `log.flags` is an [Log file content fields](https://www.elastic.co/guide/en/beats/filebeat/current/exported-fields-log.html#exported-fields-log) which is right now present in agent.yml file.

**Review by ManojS-shetty — APPROVED** [review]

**Pull request merged**